dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java

```diff
@@ -249,34 +247,6 @@ public void createServiceNoUser() throws Exception {
         Assert.assertEquals("there should be no matching service", 0, count);
     }
 
-    /**
-     * Ensures that you cannot create a service if there already exists a service with the same path
-     */
-    @Test
-    public void createServiceDuplicate() throws Exception {
```

Comment:
Refresh my memory from the presentation, why are duplicates not a concern with the new approach?

Reply:
It is more so that this test assumes that service stubs exist which is no longer the case.

Reply:
If the workflow already exists on Dockstore in the new model, one will not be created, but a new version will be added.